qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/services/snippets/qute-nested-snippets.json

```diff
@@ -0,0 +1,28 @@
+{
+    "#else" : {
+        "prefix": "else",
+		"body": [
+			"\t$2",
+			"{#else}",
+			"\t$0"
+		],
+		"description": "Else section",
+        "context": {
+            "parent": "if"
+        }
+    },
+    "#elseif": {
+		"prefix": "elseif",
+		"body": [
+			"\t$2",
+			"{#else if ${3:condition}}",
+			"\t$4",
+			"{#else}",
```

Comment:
I think we should remove this else, because if it already an else in the template, it will report an error.